cmd/node/config/lightClientP2P.toml

```
@@ -0,0 +1,89 @@
+    # If the port = 0, the node will search for a free port on the machine and use it
+    Port = "37373-38383"
+
+    # ThresholdMinConnectedPeers represents the minimum number of connections a node should have before it can start
+    # the sync and consensus mechanisms
+    ThresholdMinConnectedPeers = 3
+
+    # MinNumPeersToWaitForOnBootstrap is the minimum number of peers to wait on bootstrap or the node will wait the default
+    # time which is now set to ~20 seconds (the const defined in the common package named TimeToWaitForP2PBootstrap)
+    MinNumPeersToWaitForOnBootstrap = 10
+
+    # available transports. All defined addresses contains a single '%d' markup that is mandatory and will
+    # be replaced at runtime with the actual port value
+    [Node.Transports]
+        QUICAddress = "" # optional QUIC address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/udp/%d/quic-v1
+        WebSocketAddress = "" # optional WebSocket address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/tcp/%d/ws
+        WebTransportAddress = "" # optional WebTransport address. If this transport should be activated, should be in this format: /ip4/0.0.0.0/udp/%d/quic-v1/webtransport
+        [Node.Transports.TCP]
+            ListenAddress = "/ip4/0.0.0.0/tcp/%d" # TCP listen address
+            PreventPortReuse = false
+
+    [Node.ResourceLimiter]
+        Type = "default autoscale" #available options "default autoscale", "infinite", "default with manual scale".
+        ManualSystemMemoryInMB = 0 # not taken into account if the type is not "default with manual scale"
+        ManualMaximumFD = 0 # not taken into account if the type is not "default with manual scale"
+
+# P2P peer discovery section
+
+# The following sections correspond to the way new peers will be discovered
+# If all config types are disabled then the peer will run in single mode (will not try to find other peers)
+# If more than one peer discovery mechanism is enabled, the application will output an error and will not start
+
+[KadDhtPeerDiscovery]
+    # Enabled: true/false to enable/disable this discovery mechanism
+    Enabled = true
+
+    # Type represents the kad-dht glue code implementation.
+    # "legacy" will define the first implementation.
+    # "optimized" represents the new variant able to connect to multiple seeders at once. This implementation also has
+    # a built-in timer that will try to automatically reconnect to the seeders (in case the seeders recover after a
+    # premature shutdown)
+    Type = "optimized"
+
+    # RefreshIntervalInSec represents the time in seconds between querying for new peers
+    RefreshIntervalInSec = 10
+
+    # ProtocolIDs represents the protocols that this node will advertise to other peers
+    # To connect to other nodes, those nodes should have at least one common protocol string
+    ProtocolIDs = [
+        "/mvx/mainnet-light/v1.0.0",
```

```suggestion
        "/mvx/light/1.0.0",
```

mainnet part will be added at release, based on the chain